docker-compose.yml

```diff
@@ -8,8 +8,14 @@ services:
     ports:
       - "8080:${SERVE_PORT-8080}"
     volumes:
-      - unchained:/unchained
-      - cache:/cache
+      - type: bind
+        source: /Users/user/Data/cache
```

Comment:
If we put `docker` back in the instructions, we should adjust this. Plus, does this need a comment saying they can use any path but it should agree?